recipe/meta.yaml

```diff
@@ -29,157 +30,174 @@ source:
     - 0018-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
 
 build:
-  merge_build_host: true  # [win]
-  number: 2
-  rpaths:
-    - lib/R/lib/
-    - lib/
-  no_link:
-    # These files are modified by R CMD javareconf and can
-    # point to different Java implementations in each env.
-    - lib/R/etc/Makeconf
-    - lib/R/etc/ldpaths
-    - lib/R/etc/javaconf
-    - lib/R/etc/x64/Makeconf
-    - lib/R/etc/x64/ldpaths
-    - lib/R/etc/x64/javaconf
-  detect_binary_files_with_prefix: true
+  number: {{ build_num }}
```

Comment:
Don't use Jinja here, just omit it later in `r-base` and then the parent build number will be used. Otherwise our bots don't increment the `build_num` anymore.